wakatime/riscv64.patch

```diff
@@ -0,0 +1,33 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -14,10 +14,18 @@ license=('BSD')
+ depends=('glibc')
+ makedepends=('git' 'go')
+ checkdepends=('svn')
+-source=("${pkgname}-cli::git+https://github.com/${pkgname}/${pkgname}-cli#tag=v${pkgver}")
+-sha256sums=('SKIP')
++source=("${pkgname}-cli::git+https://github.com/${pkgname}/${pkgname}-cli#tag=v${pkgver}"
++        "main_test-arch.patch::https://github.com/wakatime/wakatime-cli/pull/642.patch")
++sha256sums=('SKIP'
++            'eeea51959573aba7091357fa428fb60a90cc4148b788cc5d8d12f4adb4753461')
+ 
+-_binname="${pkgname}-cli-linux-amd64"
++_binname="${pkgname}-cli-linux-${CARCH}"
```

Comment:
This won't work for x86_64, so I think directly patching it to `riscv64` may be more clear.